src/python/PSetTweaks/WMTweak.py

```diff
@@ -7,10 +7,7 @@
 process/config but does not depend on any CMSSW libraries. It needs to stay like this.
 
 """
-from __future__ import print_function, division
-
-from builtins import map, range, str, object
```

Comment:
Kenyi, just a random question. Why are you removing the python3 code modernization changes here?

Reply:
@amaltaro It was an unintentional removal ;). My changes were based on a version that dates before merging the python3 code modernization, apparently. I will have that updated eventually.